elastic/kibana
# [RAM] Add modal when disabling rule to prompt user to untrack alerts

## Summary
Adds a modal to all of the stack management disable flows to ask the user if they want to untrack orphaned alerts
- Rules list -> bulk disable
- Rules list -> State dropdown
- Rules list -> Action items dropdown
- Rule details

### To Test
1. Navigate to rules list or rule details
2. Try to create a rule that generate alerts
3. Disable the rule, use the modal to determine if the alerts should be untracked
4. Check the alerts to see if they were untracked or still tracked (depending on step 3)

![image](https://github.com/elastic/kibana/assets/74562234/1d980f51-d243-4742-b856-12a58369c0f9)

### Checklist
